src/UI/preferences/FlyViewSettings.qml

```diff
@@ -179,6 +180,14 @@ SettingsPage {
             enabled:            _virtualJoystick.rawValue
             fact:               _virtualJoystickAutoCenterThrottle
         }
+
+        FactCheckBoxSlider {
+            Layout.fillWidth:   true
+            text:               qsTr("Left-Handed Mode")
```

Comment:
I'm not sure people will understand what this means. I think something like "Left-Handed Mode (swap sticks)" would be better.

Reply:
I agree with that.